CHANGELOG.md

```diff
@@ -0,0 +1,110 @@
+# Changelog
+
+
+
+## [Unreleased]
```

Comment:
Okay, but can you please describe how would the updating process go? And again, these instructions should go in the CONTRIBUTING.md file as well.